src/blas/backends/mklgpu/CMakeLists.txt

```diff
@@ -44,7 +45,6 @@ set_target_properties(${LIB_OBJ} PROPERTIES
 )
 target_link_libraries(${LIB_NAME} PUBLIC ${LIB_OBJ})
 
-#Set MKL libraries as not transitive for dynamic
```

Comment:
I thought it makes sense to comment it here, but if you think it's clear why we are doing next three lines I think it's fine.

Reply:
Here, I was actually testing something. I will undelete it.